### PR TITLE
Ghetto Armoury

### PR DESCRIPTION
**Pull request by oisin100**
Adds #232

Current items:

Ductape: Can be found in the tool storage vendomat and spawns in the maints

Most glass weapons will have 10 uses before they have a chance of smashing. When a glass weapon smashes they are destroyed completely but may embed a glass shard into the target.

Glass Shank: Does 5 damage. Can be crafted with 1 glass shard and 1 ductape

Glass sword: Does 15 Damage. Can be crafted with 1 glass shard, 1 ductape, 2 Glass sheetsand 1 metal rod

Black Glass Sword: Does 15 damage. Crafted the same way glass swords are crafted but requires the mimes crayon

Refined glass sword: Does 17 damage. Can be crafted with 1 Black Glass Sword, 1 ductape, 2 metal sheets, 3 Reinforced glass and 2 metal rods. Will not shatter. 

Pneumatic spear: Does 20 throw damage by default but when you add a gas tank it does up to 10 Extra damage. throwforce = 20+tankpressure/50

So a tank with 500 Kpi would result in the spear doing 30 Damage when thrown.

Can be crafted with: 1 spear, 2 ductape and 1 transfer valve

Robust Coat: Provides 25% melee resistance. Can be crafted with 1 Leather overcoat,2 Torn cloth, 2 Sheets of metal And 2 ductape

Armored Suit: Provides 10% melee resistance. Can be crafted with 1 Black Jumpsuit,  1 Torn cloth, 1 Metal sheet And 1 ductape

Feedback is appreciated.

Credit goes to @Kmc2000 for the idea and the sprites

### CODE REVIEW

**Comment by ghost** [comment]
The armor values are too much.

The robust coat provides as much melee resistance as sec bodyarmor, and provides it in more areas than the body armor, since it's an overcoat it covers the legs.

The robust coat should provide maximum 9% melee resistance, since it covers from arms to torso to legs.

The armored suit should provide 5% melee resistance, less than the sec jumpsuit, which it is currently a duplicate of.

Glass shank literally does the same damage as the shard.

Pneumatic spear is way too fucking strong. Drop the damage to 24, the same damage as the fire axe, when fired, and keep throwdmg at 19.

Refined glass sword  is too strong. Almost as strong as a spear but onehanded. 

Glass sword is way too strong. Broken shards should not embed, embedding is too strong-instead on shatter they should do +5 brute damage in addition to the initial hit, and apply bleed.

**Comment by Kmc2000** [comment]
"Glass shank literally does the same damage as the shard."
That's the point, it's meant to be a shard that won't cut you

As far as damage and armour goes, i see your point

**Comment by oisin100** [comment]
The refined sword is such a dick to make i mean theres only 1 mime crayon on the station. yea sure it does almost as much damage as a dual-wield spear but you can make a spear so easily.

As for the embedding i could lower the embed chance more and add bleeding. It makes sense that when a glass sword shatters on your your gonna get some shards stuck in you. I could drop it down to something like 12 Brute damage per hit but any less than that and you mayaswell just make a spear.

**Comment by ghost** [comment]
that's the point

spear > all this shit

**Comment by DavidEdwards** [comment]
Looks good, but will definitely need some balancing.

**Comment by oisin100** [comment]
Dropped airspear default throw damage to 19 and maxed its gas propelled damage to 24. It also released 1/7th of its gas contents into the air when thrown.

Glass sword damage reduced to 13 And refined glass sword reduced to 16.

Glass weapons now only have a 10% chance of imbed but also have a 60% chance of bleed.

**Comment by oisin100** [comment]
Not [WIP] Anymore.

**Comment by ghost** [comment]
Have armor values been nerfed sufficiently, because again, these 'ghetto' armor values are as strong as actual sec bodysuits/gear.

Albiet if we could buff armor altogether that'd be real nice.

**Comment by oisin100** [comment]
They have been nerfed to the values you asked for Adam

File: code/modules/mob/living/carbon/update_icons.dm [hunk 1]
```diff
@@ -49,7 +49,11 @@
...
 		if(client && hud_used)
 			client.screen += r_hand
-		var/t_state = r_hand.item_state
+		var/t_state = r_hand.r_item_state
+		if(!t_state)
+			t_state = r_hand.item_state
+		if(!t_state)
+			t_state = r_hand.icon_state
 		if(!t_state)
 			t_state = r_hand.icon_state
```
Comment (X-TheDark) [inline code/modules/mob/living/carbon/update_icons.dm]: 1. Same if repeated twice.
2. Would it be possible to replace the ifs with t_state = r_hand.item_state || r.hand.icon_state ?

Comment (oisin100) [inline code/modules/mob/living/carbon/update_icons.dm]: 1. Fuck knows how that got there 0_o Ill remove it.

Edit: sorry for the delay. My github is bugging..

**Comment by Yackitori** [comment]
It'd be interesting if duct tape could be used to tape someone's mouth when aiming for the mouth, rendering them unable to speak. This will obviously not work if they're wearing a helmet that covers their face, or face masks.

**Comment by oisin100** [comment]
Could be interesting... Also maybe the ability to duct tape your limbs to help prevent zombie bites whilst at a negative side effect of making you slower.

**Comment by B1gb3ast** [comment]
should you make a ducttape a seperate item merge in and off itself lmao.

**Comment by AsV9** [comment]
I'll take my chances.

**Comment by DavidEdwards** [comment]
Did you test this AsV? Theres a lot in here.

**Pull request merged**